snuba/web/rpc/common/aggregation.py

```diff
@@ -46,7 +46,7 @@ class ExtrapolationContext(ABC):
     sample_count: int
 
     @property
-    def extrapolated_data_present(self) -> bool:
+    def data_present(self) -> bool:
```

Comment:
nit:  should this be renamed for `is_data_present` since it returns an answer to this question?